about.html

```diff
@@ -32,7 +32,7 @@ <h3>Codezix</h3>
             <!-- column 1 -->
             <p>
             <h2><strong>About me</strong></h2><br>
-            for the love of codes i build things.<br>
+            <h4>for the love of codes i build things.<h4><br>
             -----------------------------------------
```

Comment:
Is this line here on purpose?

Reply:
yea, i put it there because i wanted to include more text there. used it to signify more text.